homeassistant/components/panasonic_viera/config_flow.py

```diff
@@ -0,0 +1,179 @@
+"""Config flow for Panasonic Viera TV integration."""
+from functools import partial
+import logging
+from urllib.request import URLError
+
+from panasonic_viera import TV_TYPE_ENCRYPTED, RemoteControl, SOAPError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PIN, CONF_PORT
+from homeassistant.core import callback
+
+from .const import (  # pylint: disable=unused-import
+    CONF_APP_ID,
+    CONF_ENCRYPTION_KEY,
+    CONF_ON_ACTION,
+    DEFAULT_NAME,
+    DEFAULT_PORT,
+    DOMAIN,
+    ERROR_INVALID_PIN_CODE,
+    ERROR_NOT_CONNECTED,
+    ERROR_UNKNOWN,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config flow for Panasonic Viera."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    def __init__(self):
+        """Initialize the Panasonic Viera config flow."""
+        self._data = {
+            CONF_HOST: None,
+            CONF_NAME: None,
+            CONF_PORT: None,
+            CONF_ON_ACTION: None,
+        }
+
+        self._remote = None
+
+        self._errors = None
```

Comment:
Remove the errors instance attribute. `errors` should be a local variable to each step.

Reply:
I did that (!!!), but I screwed everything up while trying to rebase, now I'm trying to recover the old commits

Reply:
Oh, ok. Take your time. :+1: